airbyte-ci/connectors/pipelines/pipelines/commands/airbyte_ci.py

```diff
@@ -64,6 +66,27 @@ def get_latest_version() -> str:
     raise Exception("Could not find version in pyproject.toml. Please ensure you are running from the root of the airbyte repo.")
 
 
+def get_airbyte_repo() -> git.Repo:
+    """Get the airbyte repo."""
+    return git.Repo(search_parent_directories=True)
+
+
+def get_airbyte_repo_path_with_fallback() -> Path:
+    """Get the airbyte repo path."""
+    try:
+        return get_airbyte_repo().working_tree_dir
+    except git.exc.InvalidGitRepositoryError:
+        logging.warning("Could not find the airbyte repo, falling back to the current working directory.")
+        path = Path.cwd()
+        logging.warning(f"Using {path} as the airbyte repo path.")
+        return path
+
+
+def set_working_directory_to_root() -> None:
+    """Set the working directory to the root of the airbyte repo."""
+    os.chdir(get_airbyte_repo_path_with_fallback())
```

Comment:
nit: if we are changing the CWD, we should probably log something, like "Using /path/to/root as working directory"